mhs/spineroutelookup/lookup/sds_connection_factory.py

```diff
@@ -5,12 +5,15 @@
 
 import definitions
 
+_MAX_RETRIES = 3
+_CONNECTION_TIMEOUT = 5
```

Comment:
in such case it's worth adding a comment saying the unit of time span used

Reply:
Good idea!